backend/poaps/admin.py

```
@@ -213,9 +207,9 @@ class PoapDropAdmin(CloudinaryUploadMixin, admin.ModelAdmin):
     list_filter = ('status', 'event_start_at')
     search_fields = ('title', 'description', 'slug', 'legacy_poap_id')
     prepopulated_fields = {'slug': ('title',)}
-    readonly_fields = ('created_at', 'updated_at', 'artwork_public_id')
+    readonly_fields = ('claims_link', 'created_at', 'updated_at', 'artwork_public_id')
     autocomplete_fields = ('created_by',)
-    inlines = [PoapDistributionInline, PoapClaimInline]
+    inlines = [PoapDistributionInline]
```

Ruff (0.15.15)

[warning] Mutable default value for class attribute

(RUF012)